composer.json

```diff
@@ -20,15 +20,15 @@
     ],
     "require": {
         "php": ">=7.2.5",
-        "symfony/config": "^4.4 || ^5.3",
-        "symfony/dependency-injection": "^4.4 || ^5.3",
-        "symfony/http-kernel": "^4.4 || ^5.3",
+        "symfony/config": "^4.4 || ^5.3 || ^6.0",
+        "symfony/dependency-injection": "^4.4 || ^5.3 || ^6.0",
+        "symfony/http-kernel": "^4.4 || ^5.3 || ^6.0",
         "toin0u/digitalocean-v2": "^4.3"
     },
     "require-dev": {
         "guzzlehttp/promises": "^1.4",
         "nyholm/psr7": "^1.4",
-        "symfony/http-client": "^4.4 || ^5.3",
+        "symfony/http-client": "^4.4 || ^5.3 || ^6.0",
         "symfony/phpunit-bridge": "^5.3"
```

Comment:
This package can be upgraded to 6.0+ too (no need to support v5 of this one, it's a special package backward compatible with very old Symfony versions).

Reply:
Someone asked this today at Symfony slack why this was closed - and for this comment I cannot really say why this was closed. What I could see to change is hat `^5.3` to `^5.4` - but you could be clear what is missing and why.

Reply:
Oh, I think I figured out what you mean with that - Created new PR to fix that - https://github.com/dunglas/DunglasDigitalOceanBundle/pull/3